Decred Wallet/Custom Views/FloatingLabelTextField.swift

```diff
@@ -0,0 +1,119 @@
+//
+//  FloatingLabelTextField.swift
+//  Decred Wallet
+//
+//  Created by Sprinthub on 15/08/2019.
```

Comment:
Copyright header should follow this pattern: https://github.com/raedahgroup/dcrios/blob/master/Decred%20Wallet/AppDelegate.swift#L5-L7

Reply:
resolved in e7f7df7

Reply:
This isn't resolved. Copyright isn't yet as expected. To prevent ambiguity, here's what's expected:
```
// Copyright (c) 2018-2019 The Decred developers
// Use of this source code is governed by an ISC
// license that can be found in the LICENSE file.
```